pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -565,7 +565,21 @@ var _ = Describe("Converter", func() {
   </cpu>
   <vcpu placement="static">1</vcpu>
   <iothreads>3</iothreads>
-</domain>`, domainType)
+</domain>`, domainType, "%s")
+		var convertedDomainWith5Period = fmt.Sprintf(convertedDomain,
+			`<memballoon model="virtio">
+      <stats period="5"></stats>
+    </memballoon>`)
+		var convertedDomainWith0Period = fmt.Sprintf(convertedDomain,
+			`<memballoon model="virtio">
+      <stats period="0"></stats>
```

Comment:
According to the [schema for `<memballoon>`](https://gitlab.com/libvirt/libvirt/-/blob/master/docs/schemas/domaincommon.rng#L4460), the period attribute takes a positiveInteger, that is, a value which is strictly greater than 0: disabling stats collection is achieved by *not having* the `<stats>` element at all.

libvirt will still accept a period attribute equal to zero and behave as if the `<stats>` element were not present in that case, but it would be better if it was not there in the first place.

Reply:
Right, I will align that.